### PR TITLE
Fix flush_after_seconds and parse time in emit sources.

### PR DESCRIPTION
**Pull request by gtopper**

### CODE REVIEW

File: storey/sources.py [hunk 1]
```diff
@@ -42,10 +43,25 @@ def _set_error(self, ex):
...
+        return obj
+    elif isinstance(obj, float) or isinstance(obj, int):
+        return datetime.fromtimestamp(obj, tz=pytz.utc)
+    elif isinstance(obj, str):
+        if time_format is None:
+            return datetime.fromisoformat(obj)
+        else:
+            return datetime.strptime(obj, time_format)
+    else:
+        raise ValueError(f"Could not parse '{obj}' as a time.")
```
Comment (urihoenig) [inline storey/sources.py]: I guess that you wanted to print type(obj) and not obj

File: tests/test_flow.py [hunk 2]
```diff
@@ -352,6 +354,41 @@ def test_write_parquet_read_parquet_partitioned(tmpdir):
     assert read_back_result == expected
 
 
+from unittest.mock import MagicMock
+
+
+async def async_test_write_parquet_flush(tmpdir):
+    out_dir = f'{tmpdir}/test_write_parquet_read_parquet_partitioned/{uuid.uuid4().hex}/'
+    columns = ['my_int', 'my_string']
+    target = ParquetTarget(out_dir, partition_cols='my_int', columns=columns, flush_after_seconds=0)
```
Comment (urihoenig) [inline tests/test_flow.py]: Why not test with another value of flush_after_seconds? (verify it's not flushed at first & then flushed)

File: tests/test_flow.py [hunk 3]
```diff
@@ -2426,3 +2463,67 @@ def test_csv_reader_with_none_values():
...
     assert termination_result[1].key == 'b'
     assert termination_result[1].body == ['b', True, None, math.nan, math.nan, None]
+
+
+def test_bad_time_string_input():
+    controller = build_flow([
+        SyncEmitSource(time_field='time'),
+    ]).run()
+
+    try:
```
Comment (urihoenig) [inline tests/test_flow.py:2473]: You can use with pytest.raises

File: tests/test_flow.py [hunk 4]
```diff
@@ -2426,3 +2463,67 @@ def test_csv_reader_with_none_values():
...
+    finally:
+        controller.terminate()
+
+
+def test_bad_time_input():
+    controller = build_flow([
+        SyncEmitSource(time_field='time'),
+    ]).run()
+
+    try:
```
Comment (urihoenig) [inline tests/test_flow.py:2485]: with pytest.raises here too

File: storey/sources.py (hunk 1 above)
Comment (gtopper) [reply storey/sources.py]: I'll do both.

**Review by urihoenig — APPROVED** [review]

**Pull request merged**